<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakeRequestWindow</class>
 <widget class="QDialog" name="MakeRequestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Make a Request</string>
  </property>

  <!-- Name -->
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
  </widget>

  <!-- Item Name -->
  <widget class="QLabel" name="itemLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Item:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="itemLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
  </widget>

  <!-- Category -->
  <widget class="QComboBox" name="categoryComboBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Food</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Clothing</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Medicine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Other</string>
    </property>
   </item>
  </widget>

  <!-- Quantity -->
  <widget class="QLabel" name="quantityLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="quantitySpinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>

  <!-- Location -->
  <widget class="QLabel" name="locationLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Location:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="locationLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
  </widget>

  <!-- Submit Button -->
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>210</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>

 </widget>
 <resources/>
 <connections/>
</ui>
